### ISSUE TITLE
Parameterize hyphen height based on lyric-size

### ISSUE THREAD

**Issue by craigsapp** [rism-ch/verovio#740]
With the new `--lyric-size` option, the font-size of the lyrics can be controlled:

Default size 4.5:
<img width="832" alt="screen shot 2018-01-30 at 4 32 03 am" src="https://user-images.githubusercontent.com/3487289/35566618-db5e0894-0576-11e8-984c-8fe85b3e2afc.png">

Minimum size 2.0:
<img width="812" alt="screen shot 2018-01-30 at 4 30 19 am" src="https://user-images.githubusercontent.com/3487289/35566613-d78ad170-0576-11e8-8472-da2b9d292818.png">

Maximum size 8.0:
<img width="980" alt="screen shot 2018-01-30 at 4 30 49 am" src="https://user-images.githubusercontent.com/3487289/35566608-d08d78a0-0576-11e8-81ae-6fbdc68938d2.png">

However, hyphens are kept to a fixed height and size for all font sizes.  Perhaps scaling the height, length and thickness of the hyphen with the same size as the font would be good (i.e., pretend the hyphen is a font character).

Maybe also adjust the thickness of the word-continuation line as well in the same way.  The small font version looks too bold.  The large font version looks ok as it is, but probably also will look ok if made thicker.

**Comment by craigsapp** [comment]
Test data:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="http://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="http://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="4.0.0">
    <meiHead>
        <fileDesc>
            <titleStmt>
                <title />
            </titleStmt>
            <pubStmt />
        </fileDesc>
        <encodingDesc>
            <appInfo>
                <application isodate="2018-01-30T04:55:02" version="2.0.0-dev-9b82ef9">
                    <name>Verovio</name>
                    <p>Transcoded from Humdrum</p>
                </application>
            </appInfo>
        </encodingDesc>
        <workDesc>
            <work>
                <titleStmt>
                    <title />
                </titleStmt>
            </work>
        </workDesc>
    </meiHead>
    <music>
        <body>
            <mdiv>
                <score>
                    <scoreDef xml:id="scoredef-0000000584071257">
                        <staffGrp xml:id="staffgrp-0000000709406311">
                            <staffDef xml:id="staffdef-0000000162660833" clef.shape="G" clef.line="2" n="1" lines="5" />
                        </staffGrp>
                    </scoreDef>
                    <section xml:id="section-0000000566460666">
                        <measure xml:id="measure-L2" n="1" type="m-1">
                            <staff xml:id="staff-L2F1N1" n="1">
                                <layer xml:id="layer-L2F1N1" n="1">
                                    <note xml:id="note-L3F1" type="qon-0 qoff-1 pname-c acc-n oct-4 b40c-2 b12c-0 " dur="4" oct="4" pname="c" accid.ges="n">
                                        <verse xml:id="verse-L3F2" n="1">
                                            <syl xml:id="syl-L3F2" type="t">this</syl>
                                        </verse>
                                    </note>
                                    <beam xml:id="beam-L4F1-L7F1">
                                        <note xml:id="note-L4F1" type="qon-1 qoff-5_4 pname-e acc-n oct-4 b40c-14 b12c-4 " dur="16" oct="4" pname="e" accid.ges="n">
                                            <verse xml:id="verse-L4F2" n="1">
                                                <syl xml:id="syl-L4F2" type="t" con="u" wordpos="t">is</syl>
                                            </verse>
                                        </note>
                                        <note xml:id="note-L5F1" type="qon-5_4 qoff-3_2 pname-g acc-n oct-4 b40c-25 b12c-7 " dur="16" oct="4" pname="g" accid.ges="n" />
                                        <note xml:id="note-L6F1" type="qon-3_2 qoff-7_4 pname-f acc-n oct-4 b40c-19 b12c-5 " dur="16" oct="4" pname="f" accid.ges="n" />
                                        <note xml:id="note-L7F1" type="qon-7_4 qoff-2 pname-e acc-n oct-4 b40c-14 b12c-4 " dur="16" oct="4" pname="e" accid.ges="n" />
                                    </beam>
                                    <note xml:id="note-L8F1" type="qon-2 qoff-3 pname-d acc-n oct-4 b40c-8 b12c-2 " dur="4" oct="4" pname="d" accid.ges="n">
                                        <verse xml:id="verse-L8F2" n="1">
                                            <syl xml:id="syl-L8F2" type="t">some</syl>
                                        </verse>
                                    </note>
                                    <note xml:id="note-L9F1" type="qon-3 qoff-4 pname-f acc-n oct-4 b40c-19 b12c-5 " dur="4" oct="4" pname="f" accid.ges="n">
                                        <verse xml:id="verse-L9F2" n="1">
                                            <syl xml:id="syl-L9F2" type="t">text</syl>
                                        </verse>
                                    </note>
                                </layer>
                            </staff>
                        </measure>
                        <measure xml:id="measure-L10" right="end" type="m--1">
                            <staff xml:id="staff-L10F1N1" n="1">
                                <layer xml:id="layer-L10F1N1" n="1">
                                    <note xml:id="note-L11F1" type="qon-4 qoff-5 pname-g acc-n oct-4 b40c-25 b12c-7 " dur="4" oct="4" pname="g" accid.ges="n">
                                        <verse xml:id="verse-L11F2" n="1">
                                            <syl xml:id="syl-L11F2" type="i" con="d" wordpos="i">tra</syl>
                                        </verse>
                                    </note>
                                    <note xml:id="note-L12F1" type="qon-5 qoff-6 pname-a acc-n oct-4 b40c-31 b12c-9 " dur="4" oct="4" pname="a" accid.ges="n">
                                        <verse xml:id="verse-L12F2" n="1">
                                            <syl xml:id="syl-L12F2" type="m" con="d" wordpos="m">la</syl>
                                        </verse>
                                    </note>
                                    <note xml:id="note-L13F1" type="qon-6 qoff-7 pname-b acc-n oct-4 b40c-37 b12c-11 " dur="4" oct="4" pname="b" accid.ges="n">
                                        <verse xml:id="verse-L13F2" n="1">
                                            <syl xml:id="syl-L13F2" type="m" con="d" wordpos="m">la</syl>
                                        </verse>
                                    </note>
                                    <note xml:id="note-L14F1" type="qon-7 qoff-8 pname-c acc-n oct-5 b40c-2 b12c-0 " dur="4" oct="5" pname="c" accid.ges="n">
                                        <verse xml:id="verse-L14F2" n="1">
                                            <syl xml:id="syl-L14F2" type="t" wordpos="t">la.</syl>
                                        </verse>
                                    </note>
                                </layer>
                            </staff>
                        </measure>
                    </section>
                </score>
            </mdiv>
        </body>
    </music>
</mei>
```

**Comment by lpugin** [comment]
Fixed e64bcc33d0c88e0d9dc9742b8d9eb3004f5f19a5
